bessctl/module.py

```diff
@@ -22,10 +22,17 @@ def __init__(self, **kwargs):
         else:
             name = None
 
-        ret = self.bess.create_module(self.__class__.__name__, name,
-                                      self.choose_arg(None, kwargs))
-
-        self.name = ret.name
+        if not _do_not_create:
+            # create an object
+            ret = self.bess.create_module(self.__class__.__name__, name,
+                                          self.choose_arg(None, kwargs))
+            self.name = ret.name
+        else:
+            # bind to a pre-existing object, check if it's real
+            assert name != None
```

Comment:
Could you add an error message to this assertion (e.g. https://github.com/NetSys/bess/blob/master/bessctl/commands.py#L92)